### DISCUSSION TITLE
why changing hlky -->sd-webui?

### DISCUSSION THREAD

**Discussion by erhb9** [hlky/stable-diffusion-webui#934] [Q&A]
What's the difference? For me the difference is that the new one doesn't work. Especially the webui.cmd file is closing instantly.

**Comment by Janvonfalken** [comment]
Follow this versions install guide exactly and it should work. Make sure there's no spaces in you Windows Username.

**Reply by erhb9** [reply]
Which one? (edit) : I see, not the guide at the main page? The one in the discussions (https://github.com/sd-webui/stable-diffusion/discussions/93), right?

**Reply by Janvonfalken** [reply]
I meant [this](https://github.com/sd-webui/stable-diffusion-webui/wiki/Installation) one. I'll have to admit that it took me about 10 hours to get SD working, so, just keep trying.
